jdbc-adapter/virtualschema-jdbc-adapter/src/test/java/com/exasol/adapter/dialects/CustomSqlGenerationVisitorTest.java

```diff
@@ -32,7 +32,7 @@ public void testSqlGenerator() throws AdapterException {
         final SqlGenerationVisitor generator = new TestSqlGenerationVisitor(
                 new ExasolSqlDialect(null, AdapterProperties.emptyProperties()), context);
         final String actualSql = node.accept(generator);
-        assertEquals(SqlTestUtil.normalizeSql(expectedSql), SqlTestUtil.normalizeSql(actualSql));
+        assertEquals(SqlNormalizer.normalizeSql(expectedSql), SqlNormalizer.normalizeSql(actualSql));
```

Comment:
Use `JSONAssert` instead.

Reply:
We discussed that it's not a right place to use  JSONAssert